server/src/main.rs

```diff
@@ -70,12 +79,15 @@ async fn main() -> anyhow::Result<()> {
     dotenv::dotenv().expect("Error initializing environment");
     initialize_logging();
 
-    let cli = Cli::from_args();
-    match cli {
-        Cli::Serve => http::run_webserver().await,
-        Cli::GenerateAssets { static_folder } => generate_assets(static_folder).await,
-        Cli::Account(command) => command.execute().await,
-        Cli::PermissionGroup(command) => command.execute().await,
+    let Cli {
+        database_url,
+        command,
+    } = Cli::from_args();
+    match command {
+        Command::Serve => http::run_webserver(database_url).await,
```

Comment:
I'm not sure if we are there yet, but do we want to rename this to API server?
We could also do this in a different PR.

Reply:
Probably different PR, because there will still need to be an HTTP interface too, so I'm not sure exactly how everything will be spun up.